src/components/buttons/button.tsx

```diff
@@ -3,6 +3,7 @@ import { colors } from '../../styles/colors';
 import { baseButtonStyles, buttonStyles } from './buttons-styles';
 import './button.css';
 import { Separator } from '../separator/separator';
+import { LargeLabel } from '../../styles/typography/label';
 
 interface ButtonProps {
   children: React.ReactNode;
```

Comment:
poderia mudar essa prop para receber um `title` ao inves de um children. Ai a implementacao ficaria mais simples e evitaria voce poder passar uma `div` como filha da `LabelLarge` (que é uma tag de texto) sem querer. O que acha?

```suggestion
  title: string;
```